public/app/percona/settings/components/Advanced/Advanced.test.tsx

```diff
@@ -103,7 +103,7 @@ describe('Advanced::', () => {
   it('Sets correct URL from browser', () => {
     const oldLocation = window.location;
 
-    delete window.location;
+    delete (window as any).location;
```

Comment:
Do we need `as any` to access location? I see it done in other parts of the code without it

Reply:
The problem is not the access, but the delete itself.
Because `location` is not an optional prop, typescript doesn't let you delete it :S

Reply:
There's something similar at `playlist_srv.test.ts`